orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -10295,6 +10325,31 @@ definitions:
     required:
     - bandwidth_mhz
     type: object
+  network_sentry_config:
+    description: Sentry.io configuration
+    properties:
+      sample_rate:
+        example: 0.5
+        format: float
+        maximum: 1
+        minimum: 0
+        type: number
+      upload_mme_log:
+        example: false
+        type: boolean
+      url_native:
+        example: https://examplePublicKey@o0.ingest.sentry.io/0
+        format: uri
+        minLength: 0
+        type: string
+        x-nullable: false
+      url_python:
```

Comment:
Let's remove this for now? So that we don't have a field that doens't do anything.

Reply:
I think it's okay to keep it... Once we have a cross-service version of SentryConfig, we'll need to bring this field back in. The API endpoints and tests will be the same too. I think it's a nice, encapsulated config struct that needs minimal change (only on the `mconfig` side) to eliminate per-service copies. Let me know if you think otherwise.